### DISCUSSION TITLE
Outbound Call Status

### DISCUSSION THREAD

**Discussion by harish-chander** [fonoster/yaps#341] [Ideas]
Currently when making call using the callManager.call it makes a call and returns a  callID and hands off to the SIP/Media server . We will requires a async call status monitor to tell if that particular call is answered or what is the status call so we can design complex call flow or implement a call queue based setup with different polling strategies . 

Example : 
Call -> returns a Call Id
We can use a another API example callMonitor or callStatus with the call id as input to monitor and invoke extra triggers 
callManager.callStatus(CallID)-> Will Return a SIP Code (200,403,403)etc .
This is a critical feature which is will help in creating a UC/CC Based Solution on top of fonos.

**Comment by psanders** [comment]
We started some work around this. I will keep everyone posted on the progress.
